debezium-assembly-descriptors/src/main/resources/assemblies/server-distribution.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
```

Comment:
Could you share the tree output of the contents of the archive here?

Reply:
```
debezium-server/
├── CHANGELOG.md
├── conf
│   └── application.properties
├── CONTRIBUTE.md
├── COPYRIGHT.txt
├── debezium-server-1.2.0-SNAPSHOT-runner.jar
├── lib
│   ├── activation-1.1.1.jar
│   ├── annotations-2.13.13.jar
│   ├── antlr4-runtime-4.7.2.jar
│   ├── aopalliance-repackaged-2.5.0.jar
│   ├── apache-client-2.10.70.jar
│   ├── arc-1.4.1.Final.jar
│   ├── argparse4j-0.7.0.jar
│   ├── auth-2.13.13.jar
│   ├── aws-cbor-protocol-2.13.13.jar
│   ├── aws-core-2.13.13.jar
│   ├── aws-json-protocol-2.13.13.jar
│   ├── bson-3.12.3.jar
│   ├── commons-codec-1.13.jar
│   ├── commons-lang3-3.9.jar
│   ├── commons-logging-1.2.jar
│   ├── connect-api-2.5.0.jar
│   ├── connect-file-2.5.0.jar
│   ├── connect-json-2.5.0.jar
│   ├── connect-runtime-2.5.0.jar
│   ├── connect-transforms-2.5.0.jar
│   ├── debezium-api-1.2.0-SNAPSHOT.jar
│   ├── debezium-connector-mongodb-1.2.0-SNAPSHOT.jar
│   ├── debezium-connector-mysql-1.2.0-SNAPSHOT.jar
│   ├── debezium-connector-postgres-1.2.0-SNAPSHOT.jar
│   ├── debezium-connector-sqlserver-1.2.0-SNAPSHOT.jar
│   ├── debezium-core-1.2.0-SNAPSHOT.jar
│   ├── debezium-ddl-parser-1.2.0-SNAPSHOT.jar
│   ├── debezium-embedded-1.2.0-SNAPSHOT.jar
│   ├── eventstream-1.0.1.jar
│   ├── graal-sdk-19.3.1.jar
│   ├── hk2-api-2.5.0.jar
│   ├── hk2-locator-2.5.0.jar
│   ├── hk2-utils-2.5.0.jar
│   ├── httpclient-4.5.12.jar
│   ├── http-client-spi-2.13.13.jar
│   ├── httpcore-4.4.13.jar
│   ├── jackson-annotations-2.10.3.jar
│   ├── jackson-core-2.10.0.jar
│   ├── jackson-databind-2.10.0.jar
│   ├── jackson-dataformat-cbor-2.10.3.jar
│   ├── jackson-datatype-jdk8-2.10.3.jar
│   ├── jackson-datatype-jsr310-2.10.0.jar
│   ├── jackson-jaxrs-base-2.10.3.jar
│   ├── jackson-jaxrs-json-provider-2.10.3.jar
│   ├── jackson-module-jaxb-annotations-2.10.3.jar
│   ├── jakarta.activation-1.2.1.jar
│   ├── jakarta.activation-api-1.2.1.jar
│   ├── jakarta.annotation-api-1.3.5.jar
│   ├── jakarta.el-api-3.0.3.jar
│   ├── jakarta.enterprise.cdi-api-2.0.2.jar
│   ├── jakarta.inject-2.5.0.jar
│   ├── jakarta.inject-api-1.0.jar
│   ├── jakarta.interceptor-api-1.2.5.jar
│   ├── jakarta.json-1.1.6.jar
│   ├── jakarta.transaction-api-1.3.3.jar
│   ├── jakarta.validation-api-2.0.2.jar
│   ├── jakarta.ws.rs-api-2.1.5.jar
│   ├── jakarta.xml.bind-api-2.3.2.jar
│   ├── javassist-3.26.0-GA.jar
│   ├── javax.activation-api-1.2.0.jar
│   ├── javax.servlet-api-3.1.0.jar
│   ├── javax.ws.rs-api-2.1.1.jar
│   ├── jaxb-api-2.3.1.jar
│   ├── jboss-jaxb-api_2.3_spec-2.0.0.Final.jar
│   ├── jboss-jaxrs-api_2.1_spec-2.0.1.Final.jar
│   ├── jboss-logging-3.3.2.Final.jar
│   ├── jboss-logging-annotations-2.1.0.Final.jar
│   ├── jboss-logmanager-embedded-1.0.4.jar
│   ├── jboss-threads-3.1.1.Final.jar
│   ├── jersey-client-2.28.jar
│   ├── jersey-common-2.28.jar
│   ├── jersey-container-servlet-2.28.jar
│   ├── jersey-container-servlet-core-2.28.jar
│   ├── jersey-hk2-2.28.jar
│   ├── jersey-media-jaxb-2.28.jar
│   ├── jersey-server-2.28.jar
│   ├── jetty-client-9.4.24.v20191120.jar
│   ├── jetty-continuation-9.4.24.v20191120.jar
│   ├── jetty-http-9.4.24.v20191120.jar
│   ├── jetty-io-9.4.24.v20191120.jar
│   ├── jetty-security-9.4.24.v20191120.jar
│   ├── jetty-server-9.4.24.v20191120.jar
│   ├── jetty-servlet-9.4.24.v20191120.jar
│   ├── jetty-servlets-9.4.24.v20191120.jar
│   ├── jetty-util-9.4.24.v20191120.jar
│   ├── kafka-clients-2.5.0.jar
│   ├── kafka-log4j-appender-2.5.0.jar
│   ├── kafka-tools-2.5.0.jar
│   ├── kinesis-2.13.13.jar
│   ├── lz4-java-1.7.1.jar
│   ├── maven-artifact-3.6.3.jar
│   ├── microprofile-config-api-1.4.jar
│   ├── microprofile-context-propagation-api-1.0.1.jar
│   ├── microprofile-health-api-2.2.jar
│   ├── mongodb-driver-3.12.3.jar
│   ├── mongodb-driver-core-3.12.3.jar
│   ├── mssql-jdbc-7.2.2.jre8.jar
│   ├── mutiny-0.4.4.jar
│   ├── mysql-binlog-connector-java-0.19.1.jar
│   ├── mysql-connector-java-8.0.16.jar
│   ├── netty-buffer-4.1.45.Final.jar
│   ├── netty-codec-4.1.45.Final.jar
│   ├── netty-codec-dns-4.1.45.Final.jar
│   ├── netty-codec-http2-4.1.45.Final.jar
│   ├── netty-codec-http-4.1.45.Final.jar
│   ├── netty-codec-socks-4.1.45.Final.jar
│   ├── netty-common-4.1.45.Final.jar
│   ├── netty-handler-4.1.45.Final.jar
│   ├── netty-handler-proxy-4.1.45.Final.jar
│   ├── netty-nio-client-2.10.70.jar
│   ├── netty-reactive-streams-2.0.4.jar
│   ├── netty-reactive-streams-http-2.0.4.jar
│   ├── netty-resolver-4.1.45.Final.jar
│   ├── netty-resolver-dns-4.1.45.Final.jar
│   ├── netty-transport-4.1.45.Final.jar
│   ├── netty-transport-native-epoll-4.1.45.Final-linux-x86_64.jar
│   ├── netty-transport-native-unix-common-4.1.45.Final.jar
│   ├── osgi-resource-locator-1.0.1.jar
│   ├── plexus-utils-3.2.1.jar
│   ├── postgresql-42.2.12.jar
│   ├── profiles-2.13.13.jar
│   ├── protobuf-java-3.8.0.jar
│   ├── protocol-core-2.13.13.jar
│   ├── quarkus-arc-1.4.1.Final.jar
│   ├── quarkus-core-1.4.1.Final.jar
│   ├── quarkus-development-mode-spi-1.4.1.Final.jar
│   ├── quarkus-ide-launcher-1.4.1.Final.jar
│   ├── quarkus-jsonp-1.4.1.Final.jar
│   ├── quarkus-netty-1.4.1.Final.jar
│   ├── quarkus-resteasy-1.4.1.Final.jar
│   ├── quarkus-resteasy-common-1.4.1.Final.jar
│   ├── quarkus-resteasy-server-common-1.4.1.Final.jar
│   ├── quarkus-security-1.1.0.Final.jar
│   ├── quarkus-smallrye-health-1.4.1.Final.jar
│   ├── quarkus-vertx-core-1.4.1.Final.jar
│   ├── quarkus-vertx-http-1.4.1.Final.jar
│   ├── reactive-streams-1.0.3.jar
│   ├── reflections-0.9.12.jar
│   ├── regions-2.13.13.jar
│   ├── resteasy-core-4.5.3.Final.jar
│   ├── resteasy-core-spi-4.5.3.Final.jar
│   ├── sdk-core-2.13.13.jar
│   ├── slf4j-api-1.7.28.jar
│   ├── slf4j-jboss-logging-1.2.0.Final.jar
│   ├── smallrye-config-1.7.0.jar
│   ├── smallrye-config-common-1.7.0.jar
│   ├── smallrye-health-2.2.0.jar
│   ├── smallrye-health-extension-api-2.2.0.jar
│   ├── snappy-java-1.1.7.3.jar
│   ├── utils-2.13.13.jar
│   ├── validation-api-2.0.1.Final.jar
│   ├── vertx-auth-common-3.8.5.jar
│   ├── vertx-bridge-common-3.8.5.jar
│   ├── vertx-core-3.8.5.jar
│   ├── vertx-web-3.8.5.jar
│   ├── vertx-web-common-3.8.5.jar
│   ├── wildfly-common-1.5.4.Final-format-001.jar
│   └── zstd-jni-1.4.4-7.jar
├── LICENSE-3rd-PARTIES.txt
├── LICENSE.txt
├── README.md
└── run.sh
```